Documentation/gettingstarted/clustermesh/clustermesh.rst

```diff
@@ -26,6 +30,28 @@ Cluster Addressing Requirements
 * The network between clusters must allow the inter-cluster communication. The
   exact ports are documented in the :ref:`firewall_requirements` section.
 
+Additional Requirements for Native-routed Datapath Modes
+--------------------------------------------------------
+
+* Cilium in each cluster must be configured with a native routing CIDR that
+  covers all the PodCIDR ranges across all connected clusters. Cluster CIDRs are
+  typically allocated from the ``10.0.0.0/8`` private address space. When this
+  is a case a native routing CIDR such as ``10.0.0.0/9`` should cover all
+  clusters:
+
+ * ConfigMap option ``native-routing-cidr=10.0.0.0/9``
+ * Helm option ``--set nativeRoutingCIDR=10.0.0.0/9``
+ * ``cilium install`` option ``--native-routing-cidr=10.0.0.0/9``
```

Comment:
FYI, these options are deprecated and going away in 1.12, so we should probably replace with the new ones:

```suggestion
 * ConfigMap option ``ipv4-native-routing-cidr=10.0.0.0/9``
 * Helm option ``--set ipv4NativeRoutingCIDR=10.0.0.0/9``
 * ``cilium install`` option ``--ipv4-native-routing-cidr=10.0.0.0/9``
```

Reply:
Fixed by https://github.com/cilium/cilium/pull/20408